src/sentry/notifications/notifications/rules.py

```diff
@@ -166,6 +166,20 @@ def send(self) -> None:
                         recipient_email=participant.email,
                         recipient_username=participant.username,
                     )
+                if (
+                    features.has("organizations:active-release-monitor-alpha", self.organization)
+                    and last_release
+                ):
+                    ReleaseActivity.objects.create(
+                        type=ReleaseActivity.Type.issue,
+                        data={
+                            "provider": EXTERNAL_PROVIDERS[provider],
+                            "issue_id": self.group.id,
+                            "issue_error_count": self.group.times_seen,
+                            "issue_affected_users": self.group.count_users_seen(),
+                        },
+                        release=last_release,
+                    )
```

Comment:
How often will this get called? `count_users_seen` makes a snuba query, so just want to verify.

Reply:
This path is feature-flagged to our org only and only when a new issue or regression occurs in an active release window. There should be around a 2:1 with the number of times the `ActiveReleaseAlertNotification` triggers to the number of times `group.count_users_seen()` gets called.